### PR TITLE
Added newrelic recipe

### PR DESCRIPTION
**Pull request by ivoflipse**

### CODE REVIEW

File: newrelic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - libtool  # [unix]
+    - python
```
Comment (Maxyme) [inline newrelic/meta.yaml]: python should be the first requirement, then setuptools.
libtool should be pinned to version >=2.4.2 that's the on on anaconda channel
Not sure it needs setuptools to run.

File: newrelic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - setuptools
+    - toolchain
+  run:
+    - python
+    - setuptools
+
+test:
+  requires:
```
Comment (Maxyme) [inline newrelic/meta.yaml]: require python not needed for tests.

File: newrelic/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+  fn: newrelic-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/n/newrelic/newrelic-{{ version }}.tar.gz
+  sha256: 437782a9446a4d3589d9b4568cd4848411988bbbf3dd46ec92f0624e8cdc8b93
+
+build:
+  number: 0
+  script:
```
Comment (Maxyme) [inline newrelic/meta.yaml]: Have you tried making it a noarch package?

File: newrelic/meta.yaml (hunk 2 above)
Comment (ivoflipse) [inline newrelic/meta.yaml]: Removed

File: newrelic/meta.yaml (hunk 1 above)
Comment (ivoflipse) [inline newrelic/meta.yaml]: Pinned and removed.
I got the recipe from conda-forge, so I presume those linting rules are your preferences?

File: newrelic/meta.yaml (hunk 3 above)
Comment (ivoflipse) [inline newrelic/meta.yaml]: I'll give it a shot, but not sure if that works when you have entry points

File: newrelic/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  noarch: python
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - libtool >= 2.4.2  # [unix]
+    - toolchain
```
Comment (ivoflipse) [inline newrelic/meta.yaml]: I'm getting an error on Windows that `toolchain` is missing. Which seems to imply we don't have it in our channels.

Comment (Maxyme) [inline newrelic/meta.yaml]: https://anaconda.org/clinicalgraphics/toolchain/files
it seems to be here odd.

File: newrelic/meta.yaml (hunk 1 above)
Comment (Maxyme) [inline newrelic/meta.yaml]: They are the unofficial CG linting rules. Having python on top and setup tools makes it easy to see if it's a python recipe.

File: newrelic/meta.yaml (hunk 4 above)
Comment (Korijn) [inline newrelic/meta.yaml]: They're labeled as archived so you won't find them.

**Pull request merged**